test/select_tests.cpp

```diff
@@ -438,4 +438,76 @@ TEST(JoinTypes) {
 
   stmt = (SelectStatement*) result.getStatement(11);
   ASSERT_NULL(stmt->fromTable->join);
-}
\ No newline at end of file
+}
+
+TEST(SetOperations) {
+  SelectStatement* stmt;
+
+  TEST_PARSE_SQL_QUERY("select a from t1 union select b from t2; \
```

Comment:
limit/order after set operations should probably included here? As in "SELECT a FROM t1 UNION (...) ORDER BY a LIMIT 5"

Reply:
I have added such tests for `WHERE`/`LIMIT`/`ORDER` as suggested